document_url_google_drive/readme/USAGE.md

```diff
@@ -0,0 +1,8 @@
+To use this module, you need to:
+
+#. Open the form view of an object (Example: Customer Invoice INV/2019/0007). #. Go to
+the chatter and click on the attached icon. #. Click **Add GDrive link**. #. Fill the
+wizard fields and click on Add button. #. In the open window, select the files you need
+and press the select button. #. You will see a new **URL attachment** in the set of
+attachments related to the object. #. In order to log in under another google user,
+click on the logout icon located after **Add GDrive link**
```

Comment:
Can you please fix the line breaks?

Reply:
Hello @yostashiro thank you for your feedback! How exactly should they be fixed? Could you please add a suggestion so that we could accept it